examples/ai/chat/simplemind_example.py

```diff
@@ -0,0 +1,112 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "marimo",
+#     "simplemind==0.1.3",
```

Comment:
good job with the version pinning, as we're pre 1.0

Reply:
marimo does this automatically (for reproducibility!). If you make breaks, we can (or you can, if you'd like) update the example